src/router.js

```diff
@@ -107,6 +107,10 @@ const router = new Router({
 			path: '/new',
 			redirect: () => `/${getInitialView()}/now/new/${getPreferredEditorRoute()}/0/${getDefaultStartDateForNewEvent()}/${getDefaultEndDateForNewEvent()}`,
```

Comment:
I think something similar to the following might be better instead of duplicating the route (didn't test).

```suggestion
			path: '/new/:view?',
			redirect: (to) => `/${to.params.view || getInitialView()}/now/new/${getPreferredEditorRoute()}/0/${getDefaultStartDateForNewEvent()}/${getDefaultEndDateForNewEvent()}`,
```

Reply:
That's a good idea. Thanks!